.github/templates/main.yml

```diff
@@ -103,7 +103,8 @@ steps:
 #@ end
 
 #@ def buildDocs():
-#@ docsCondition = "${{ contains(github.ref, 'release') }}"
+#@ isRelease = "contains(github.head.ref, 'release')"
```

Comment:
I actually have a question. In the github context docs (https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) I can see that there is `github.head_ref`, but not `github.head.ref`. Is it documented somewhere else?

Reply:
That is, unfortunately, a typo, so it's a really good catch.
Working with release workflows is tricky because I can't really release, so I can only use another test repo for these release tests. For a modification so small I didn't use that repo, as I only had to change the name of a var in a condition and the typo happened. So, thank you.

I wish github could somehow warn of "unrecognized variables" in some way.

Reply:
Yeah, unfortunately it's almost testing in production with those kind of things 🥲